crates/l2/sequencer/l1_committer.rs

```diff
@@ -355,7 +355,8 @@ impl L1Committer {
                     &acc_privileged_txs,
                     acc_account_updates.clone().into_values().collect(),
                 )?;
-                generate_blobs_bundle(&state_diff)
+                spawned_rt::tasks::spawn_blocking(move || generate_blobs_bundle(&state_diff))
```

Comment:
Mmmhhh, can't this be avoided using [start_blocking](https://github.com/lambdaclass/spawned/blob/main/concurrency/src/tasks/gen_server.rs#L146) for the whole `GenServer` instead?